server/src/main/java/org/apache/druid/server/metrics/IndexerTaskCountStatsProvider.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.server.metrics;
+
+import java.util.Map;
+
+/**
+ * Provides task count metrics for the indexers
+ * These metrics are reported by indexers
+ */
+public interface IndexerTaskCountStatsProvider
+{
+  /**
+   * The number of running tasks on an indexer
```

Comment:
Minor suggestion for the javadocs but this can be addressed later.
```suggestion
   * Map from datasource name to the number of running tasks on the Indexer.
```